docs/acme.md

```diff
@@ -0,0 +1,89 @@
+by running:
+
+```
+$ step ca provisioner add acme --type ACME
+```
+
+Now restart or SIGHUP `step-ca` to pick up the new configuration.
+
+That’s it.
+
+## Configuring Clients
+
+To configure an ACME client to connect to `step-ca` you need to:
+
+1. Point the client at the right ACME directory URL
+2. Tell the client to trust your CA’s root certificate
+
+Once certificates are issued, you’ll also need to ensure they’re renewed before
+they expire.
+
+### Pointing Clients at the right ACME Directory URL
+
+Most ACME clients connect to Let’s Encrypt by default. To connect to `step-ca`
+you need to point the client at the right [ACME directory
+URL](https://tools.ietf.org/html/rfc8555#section-7.1.1).
+
+A single instance of `step-ca` can have multiple ACME provisioners, each with
+their own ACME directory URL that looks like:
+
+```
+https://{ca-host}/acme/{provisioner-name}/directory
+```
+
+We just added an ACME provisioner named “acme”. Its ACME directory URL is:
+
+```
+https://ca.internal/acme/acme/directory
+```
+
+### Telling clients to trust your CA’s root certificate
+
+Communication between an ACME client and server [always uses
+HTTPS](https://tools.ietf.org/html/rfc8555#section-6.1). By default, client’s
+will validate the server’s HTTPS certificate using the public root certificates
+in your system’s [default
+trust](https://smallstep.com/blog/everything-pki.html#trust-stores) store.
+That’s fine when you’re connecting to Let’s Encrypt: it’s a public CA and its
+root certificate is in your system’s default trust store already. Your internal
+root certificate isn’t, so HTTPS connections from ACME clients to `step-ca` will
+fail.
+
+There are two ways to address this problem:
+
+1. Explicitly configure your ACME client to trust `step-ca`'s root certificate, or
+2. Add `step-ca`'s root certificate to your system’s default trust store (e.g.,
+   using `[step certificate
+   install](https://smallstep.com/docs/cli/certificate/install/)`)
+
+If you’re using your CA for TLS in production, explicitly configuring your ACME
+client to only trust your root certificate is a better option. We’ll
+demonstrate this method with several clients below.
+
+If you’re simulating Let’s Encrypt in pre-production, installing your root
+certificate is a more faithful simulation of production. Once your root
+certificate is installed, no additional client configuration is necessary.
+
+> Caution: adding a root certificate to your system’s trust store is a global
+> operation. Certificates issued by your CA will be trusted everywhere,
+> including in web browsers.
+
+`step-ca` should work with any ACMEv2
+([RFC8555](https://tools.ietf.org/html/rfc8555)) compliant client that supports
+the http-01 or dns-01 challenge. If you run into any issues please let us know
+[on gitter](https://gitter.im/smallstep/community) or [in an
+issue](https://github.com/smallstep/certificates/issues/new?template=bug_report.md).
```

Comment:
Add example using certbot or the client recommended by let's encrypt.

Reply:
I added the certbot example from the blog post.